lib/sidekiq_alive/redis/base.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module SidekiqAlive
+  module Redis
+    class Base
+      def set(...)
+        raise(NotImplementedError)
+      end
+
+      def zadd(set_key, ex, key)
+        raise(NotImplementedError)
+      end
+
+      def zrange(set_key, start, stop)
+        raise(NotImplementedError)
+      end
+
+      def zrangebyscore(set_key, min, max)
+        raise(NotImplementedError)
+      end
+
+      def zrem(set_key, key)
+        raise(NotImplementedError)
+      end
+
+      def delete(key)
+        raise(NotImplementedError)
+      end
```

Comment:
For usability I'd consider making this error more specific with details on the method args, not just its name and arity.